sentences/de/_common.yaml

```diff
@@ -16,5 +16,6 @@ expansion_rules:
   area: "(in (der | dem) | im) {area}"
   turn_on: "(an | ein)"
   turn_off: "(ab | aus)"
+  cover_variants: "(das Tor | (die | alle) Tore | die Garage | (die | alle) Garagen | das Garagentor | (die | alle) Garagentore | das Rollo | (die | alle) Rollos | die Abdeckung | (die | alle) Abdeckungen)"
```

Comment:
I would also add "Rollladen" (roller shutter) here as this is probably a very common cover to control.